papers/emily_dorne/main.md

```diff
@@ -187,16 +195,16 @@ The [model experimentation](#model-experimentation) phase did not explore altern
     <th>Explanation</th>
   </tr>
   <tr>
-    <td>Filter to water area and use a larger bounding box</td>
-    <td>Land pixels are filtered out because they are usually greener than water areas, and can generate falsely high cyanobacteria estimates. Sentinel-2's scene classification band is not perfectly accurate, but is sufficient for masking non-water pixels. Since ground sampling points are often near land (taken from the shore or the dock), a larger bounding box (2,000m) is used to ensure the relevant water pixels are included.</td>
+    <td>Filter to water area and use a large bounding box</td>
```

Comment:
In context here, we aren't doing a comparison to something else so "large" feels clearer to me than "larger"